specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2019-05-06-preview/searchservice.json

```diff
@@ -3676,6 +3676,54 @@
         "url": "https://lucene.apache.org/core/4_10_3/analyzers-common/org/apache/lucene/analysis/pattern/PatternReplaceCharFilter.html"
       }
     },
+    "Similarity": {
+      "discriminator": "@odata.type",
+      "properties": {
+        "@odata.type": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "@odata.type"
+      ],
+      "description": "Abstract base class for similarity algorithms.",
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/en-us/azure/search/index-ranking-similarity"
```

Comment:
Please remove `/en-us` from doc links so that readers in other locales are redirected to localized pages.